java/code/src/com/redhat/rhn/domain/server/ServerFactory.java

```diff
@@ -261,6 +261,20 @@ public static List lookupServersWithCustomKey(Long userId, Long cikid) {
         );
     }
 
+    /**
+     * Regenerate the package needed cache for the server
+     *
+     * @param serverId the server ID
+     */
+    public static void updateServerNeededCache(Long serverId) {
```

Comment:
i'm not sure but it does not look like this expects `null` to be passed so i would just keep this `long`.

Reply:
Makes sense. Changing.